lte/protos/subscriberauth.proto

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+//
+// This is a proxy for Diamater application S6a/S6d 3GPP 29.272
+// It is watered down for use with OpenAirInterface EPC with extensibility in mind
+//
+// For details about fields see:
+// http://www.etsi.org/deliver/etsi_ts/129200_129299/129272/10.09.00_60/ts_129272v100900p.pdf
```

Comment:
this comment is a bit misleading, 129 272 V10.9.0 doesn't mention 5G. Can you remove it or point to the right RFC instead?

Reply:
I've addressed the comment.